infra/cifuzz/actions/entrypoint.py

```diff
@@ -12,46 +12,69 @@
-  repo_name = os.environ['GITHUB_REPOSITORY'].rsplit('/', 1)[-1]
-  commit_sha = os.environ['GITHUB_SHA']
+  """Runs OSS-Fuzz project's fuzzers for CI tools.
+  This script is used to kick off the Github Actions CI tool. It is the
+  entrypoint  of the Dockerfile in this directory. This action can be added to
+  any OSS-Fuzz project's workflow that uses Github.
+
+  Required environment variables:
+    PROJECT_NAME: The name of OSS-Fuzz project.
+    FUZZ_TIME: The length of time in seconds that fuzzers are to be run.
+    GITHUB_REPOSITORY: The name of the Github repo that called this script.
+    GITHUB_SHA: The commit SHA that triggered this script.
+
+  Returns:
+    0 on success or 1 on Failure.
+  """
+  oss_fuzz_project_name = os.environ.get('PROJECT_NAME')
+  fuzz_seconds = int(os.environ.get('FUZZ_SECONDS'))
+  github_repo_name = os.path.basename(os.environ.get('GITHUB_REPOSITORY'))
+  commit_sha = os.environ.get('GITHUB_SHA')
+
+  # Get the shared volume directory and create required directorys.
+  workspace = os.environ.get('GITHUB_WORKSPACE')
+  if not workspace:
+    logging.error('This script needs to be run in the Github action context.')
+    return 1
+  git_workspace = os.path.join(workspace, 'storage')
+  os.makedirs(git_workspace, exist_ok=True)
+  out_dir = os.path.join(workspace, 'out')
+  os.makedirs(out_dir, exist_ok=True)
 
   # Build the specified project's fuzzers from the current repo state.
-  print('Building fuzzers\nproject: {0}\nrepo name: {1}\ncommit: {2}'.format(
-      project_name, repo_name, commit_sha))
-  command = [
-      'python3', '/src/oss-fuzz/infra/cifuzz.py', 'build_fuzzers', project_name,
-      repo_name, commit_sha
-  ]
-  print('Running command: "{0}"'.format(' '.join(command)))
-  try:
-    subprocess.check_call(command)
-  except subprocess.CalledProcessError as err:
-    sys.stderr.write('Error building fuzzers: "{0}"'.format(str(err)))
-    return err.returncode
+  if not cifuzz.build_fuzzers(oss_fuzz_project_name, github_repo_name,
+                              commit_sha, git_workspace, out_dir):
+    logging.error('Error building fuzzers for project %s.',
+                  oss_fuzz_project_name)
+    return 1
 
   # Run the specified project's fuzzers from the build.
-  command = [
-      'python3', '/src/oss-fuzz/infra/cifuzz.py', 'run_fuzzers', project_name
-  ]
-  print('Running command: "{0}"'.format(' '.join(command)))
-  try:
-    subprocess.check_call(command)
-  except subprocess.CalledProcessError as err:
-    sys.stderr.write('Error running fuzzers: "{0}"'.format(str(err)))
-    return err.returncode
-  print('Fuzzers ran successfully.')
+  run_status, bug_found = cifuzz.run_fuzzers(oss_fuzz_project_name,
+                                             fuzz_seconds, out_dir)
+  if not run_status:
+    logging.error('Error occured while running fuzzers for project %s.',
+                  oss_fuzz_project_name)
+    return 1
+  if bug_found:
+    logging.debug('Bug found.')
```

Comment:
nit: I think you use the "debug" level more than you should. For example, this feels more like info.
Here's a guide on which levels should be used when 
 https://docs.python.org/2/howto/logging.html#when-to-use-logging

Reply:
Fixed.